sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -631,71 +665,47 @@ def query_items_change_feed( # pylint: disable=unused-argument
             ~datetime.datetime: processing change feed from a point of time. Provided value will be converted to UTC.
             By default, it is start from current ("Now")
         :type start_time: Union[~datetime.datetime, Literal["Now", "Beginning"]]
-        :keyword Literal["High", "Low"] priority: Priority based execution allows users to set a priority for each
+        :keyword priority: Priority based execution allows users to set a priority for each
             request. Once the user has reached their provisioned throughput, low priority requests are throttled
             before high priority requests start getting throttled. Feature must first be enabled at the account level.
+        :type priority: Optional[Literal["High", "Low"]]
+        :keyword change_feed_mode: The change feed mode enum to use when processing change feed items.
+            LATEST_VERSION: Query latest items from 'start_time' or 'continuation' token.
+            ALL_VERSIONS_AND_DELETES: Query all versions and deleted items from either 'fromNow'(default)
+                                      or 'continuation' token.
+        :type change_feed_mode: Optional[ChangeFeedMode]
         :keyword Callable response_hook: A callable invoked with the response metadata.
         :param Any args: args
         :returns: An Iterable of items (dicts).
         :rtype: Iterable[Dict[str, Any]]
         """
         # pylint: disable=too-many-statements
-        if kwargs.get("priority") is not None:
-            kwargs['priority'] = kwargs['priority']
+        add_args_to_kwargs(args, kwargs)
```

Comment:
I don't believe this is needed here, as I don't think the async version ever supported `args` to begin with, so the backcompat shouldn't be necessary.